conbench/templates/app.html

```diff
@@ -138,6 +138,32 @@ <h4 class="modal-title"><span class="glyphicon glyphicon-trash submenu"></span>&
     });
 }
 
+/*
+INFO:
+    Adds free text search ability for specific columns, based on selected column indexes.
+    Based on the indexes of the columns, input boxes are created. The search is triggered on
+    every key press.
+INPUTS:
+ data_table_id: string
+ column_indexes: array
+ */
+function free_text_column_search(data_table_id, column_indexes) {
+    $(document).ready(function() {
+        var table = $(data_table_id).DataTable();
+        table.columns(column_indexes).every(function() {
+            var that = this;
+            var title = this.header().textContent;
+            $('<input type="text" placeholder="Search ' + title + '" />')
+                .appendTo(this.header())
+                .on('keyup change clear', function() { //column search/sorting triggered on every input
```

Comment:
I like this choice! I don't think it hits the API every keystroke; it searches through the 1000 results already in-memory.

Reply:
great :)